build/make/olm.mk

```diff
@@ -20,6 +20,7 @@ generate_olm_bundle_yaml: _check_operator_sdk_version _generate_olm_deployment_f
 		--channels fast \
 		--metadata && \
 	mv bundle.Dockerfile build/
+	find deploy/bundle/manifests -name \*.yaml -exec yq --indentless -iYw 1000000000 . {} \;
```

Comment:
Yeah those changes are what we're trying to undo. I'll look into what's going on here :thinking:

Reply:
Hmm, I tested the command in an `ubuntu 20.04` & `make 4.2.1` container environment and it worked without problem.

Adding double quotes around the pattern results in no matches since we no longer need `\*`. Can you check if 4fac554d fixes the issue?

Reply:
Still does not work. what is your yq version? mine is `https://github.com/kislyuk/yq` 2.10.1.

```
> find deploy/bundle/manifests -name \*.yaml -exec echo {} \;   
deploy/bundle/manifests/workspace.devfile.io_devworkspaces.yaml
deploy/bundle/manifests/workspace.devfile.io_devworkspacetemplates.yaml
deploy/bundle/manifests/devworkspace-controller-view-workspaces_rbac.authorization.k8s.io_v1_clusterrole.yaml
deploy/bundle/manifests/devworkspace-controller-edit-workspaces_rbac.authorization.k8s.io_v1_clusterrole.yaml
deploy/bundle/manifests/devworkspace-controller-manager-service_v1_service.yaml
deploy/bundle/manifests/controller.devfile.io_devworkspaceroutings.yaml
deploy/bundle/manifests/devworkspace-controller-manager-metrics-service_v1_service.yaml
deploy/bundle/manifests/devworkspace-controller-serviceaccount_v1_serviceaccount.yaml
deploy/bundle/manifests/devworkspace-operator.clusterserviceversion.yaml
deploy/bundle/manifests/devworkspace-controller-configmap_v1_configmap.yaml
deploy/bundle/manifests/devworkspace-controller-metrics-reader_rbac.authorization.k8s.io_v1_clusterrole.yaml
```
then I played with yq and found that the following works for me, I have no idea why it matters:
```make
	find deploy/bundle/manifests -name '*.yaml' -exec yq --indentless -w 1000000000 -iY . {} \;
```

But it can't be so simple ) My command still lead to 
```diff
diff --git a/deploy/bundle/manifests/devworkspace-operator.clusterserviceversion.yaml b/deploy/bundle/manifests/devworkspace-operator.clusterserviceversion.yaml
index 0e3e6691..99777d4c 100644
--- a/deploy/bundle/manifests/devworkspace-operator.clusterserviceversion.yaml
+++ b/deploy/bundle/manifests/devworkspace-operator.clusterserviceversion.yaml
@@ -406,9 +406,9 @@ spec:
     - v1
     containerPort: 443
     conversionCRDs:
-    - devworkspaces.workspace.devfile.io
+    - devworkspacetemplates.workspace.devfile.io
     deploymentName: devworkspace-controller-manager
-    generateName: cdevworkspaces.kb.io
+    generateName: cdevworkspacetemplates.kb.io
     sideEffects: None
     targetPort: conversion
     type: ConversionWebhook
@@ -417,9 +417,9 @@ spec:
     - v1
     containerPort: 443
     conversionCRDs:
-    - devworkspacetemplates.workspace.devfile.io
+    - devworkspaces.workspace.devfile.io
     deploymentName: devworkspace-controller-manager
-    generateName: cdevworkspacetemplates.kb.io
+    generateName: cdevworkspaces.kb.io
     sideEffects: None
     targetPort: conversion
     type: ConversionWebhook
```

Reply:
And some day we need PR check to ensure up to date but I'm giving up with https://github.com/devfile/devworkspace-operator/pull/449. Maybe you will have an idea what is wrong?

Reply:
The error you're running into is actually unrelated -- https://github.com/operator-framework/operator-sdk/issues/5022

I'll look into your suggestion, though I'm still not clear on what the failure is caused by.

I'm on `yq 2.12.0` here

Reply:
Added workaround for the operator-sdk issue, webhooks order should be stable now.

Reply:
works for me now